sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -88,6 +89,10 @@ class DynamoDBOnlineStore(OnlineStore):
     _dynamodb_resource = None
     _aioboto_session = None
 
+    @property
+    def async_supported(self) -> SupportedAsyncMethods:
+        return SupportedAsyncMethods(read=True)
```

Comment:
an online store can opt in to using async read/write calls as they become available.